README.md

```diff
@@ -549,6 +549,26 @@ Here are the metrics settings:
     "vm.max_map_count" = "262144"
     ```
 
+#### Boot-related settings for bare metal variants
```

Comment:
Once again, for the short term this will be only on metal, but longer term probably not.

We should talk a bit more about how is the best way to do this.